documentation/modules/proc-configure-openid-connect-for-kubernetes.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// assembly-configuring.adoc
+
+[id='config-openid-connect-for-kubernetes']
+= Configuring EnMasse Console to use OpenID Connect
+
+In order to use the EnMasse Console on Kubernetes, you must configure Kubernetes to use
+https://kubernetes.io/docs/reference/access-authn-authz/authentication/#openid-connect-tokens[OpenID Connect (OIDC)]
+as an Authentication Strategy.  Then you must create a `consoleservice` resource refering to your OIDC Provider.
+
+.Prerequestises
+
+Before you begin you need to know the following details from your OpenID Connect provider:
+
+- OIDC Discovery URL
+- OIDC scopes
+- client id
+- client secret
+
+The Kubernetes api-server must be configured to use the
+https://kubernetes.io/docs/reference/access-authn-authz/authentication/#configuring-the-api-server[OpenID Connect plugin].
+If you are using minikube https://minikube.sigs.k8s.io/docs/tutorials/openid_connect_auth/[these instructions]
+will guide you.
+
+The correct setting for OIDC scopes varies depending on the OIDC provider.  Refer to your OIDC Provider's instructions.
```

Comment:
I would move this sentence and the one that follows under the first bullet point (since it talks about needing the OIDC scope)